pkg/monitor/cluster/jobconditions.go

```diff
@@ -0,0 +1,54 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	batchv1 "k8s.io/api/batch/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/Azure/ARO-RP/pkg/util/namespace"
+)
+
+var jobConditionsExpected = map[batchv1.JobConditionType]v1.ConditionStatus{
+	batchv1.JobComplete: v1.ConditionTrue,
+	batchv1.JobFailed:   v1.ConditionFalse,
+}
+
+func (mon *Monitor) emitJobConditions(ctx context.Context) error {
+
+	jobs, err := mon.cli.BatchV1().Jobs("").List(ctx, metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	for _, job := range jobs.Items {
+		if !namespace.IsOpenShift(job.Namespace) {
```

Comment:
This returns true if namespace == default.  So we wouldn't be emitting metrics for this namespace. Do we not want to do that?

Reply:
thanks for the comment.
The way it is done here is consisten with the other metrics emission and the default namespace get the metrics too. I have tested by changing the namespace in the attached unit test just now and the metric gets emitted, so I think we are covered here.